vite.config.ts

```diff
@@ -11,4 +12,17 @@ export default defineConfig({
       '@': fileURLToPath(new URL('./src', import.meta.url)),
     },
   },
+  build: {
+    terserOptions: {
+      compress: {
+        // Prevent Infinity from being compressed to 1/0, may cause performance issues on Chrome
+        keep_infinity: true,
+        drop_console: VITE_DROP_CONSOLE,
```

Comment:
Just set it to true

Reply:
https://github.com/Lipoic/Lipoic-Frontend/pull/35/commits/d40e8415b5dc066af086737a87b8fc1ca6270039